extensions/figma-files/package.json

```diff
@@ -5,35 +5,42 @@
   "description": "Lists Figma files allowing you to search and navigate to them.",
   "icon": "command-icon.png",
   "author": "michaelschultz",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "license": "MIT",
   "commands": [
     {
       "name": "index",
       "title": "Search Files",
       "subtitle": "Figma",
       "description": "Lists Figma files allowing you to search and navigate to them.",
-      "mode": "view",
-      "preferences": [
-        {
-          "name": "PERSONAL_ACCESS_TOKEN",
-          "type": "password",
-          "required": true,
-          "title": "Personal Access Token",
-          "description": "Log in to Figma and click settings to setup a token"
-        },
-        {
-          "name": "TEAM_ID",
-          "type": "password",
-          "required": true,
-          "title": "Team ID",
-          "description": "Find this in the url after navigating to your team page"
-        }
-      ]
+      "mode": "view"
+    },
+    {
+      "name": "menu-bar",
+      "title": "Quicklook",
+      "subtitle": "Figma",
+      "description": "See your Figma most recent files at a glance",
+      "mode": "menu-bar"
```

Comment:
might make sense to enable [background refresh](https://developers.raycast.com/information/background-refresh) for this command, so it's always up to date? 🤔 could refresh every couple of hours or so